### PR TITLE
material-ui/core from 4.12.3 to 4.12.4

### PR DESCRIPTION
**Pull request by Harshit-Dandriyal**
Signed-off-by: Harshit Dandriyal <56961488+Harshit-Dandriyal@users.noreply.github.com>

**Description**
Our Provided UI is now in material-UI/core 4.12.4

This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: provider-ui/pages/_app.js [hunk 1]
```diff
@@ -229,7 +229,6 @@ class MesheryProviderApp extends App {
           </Head>
           <MuiThemeProvider theme={theme}>
             <div className={classes.root}>
-              <CssBaseline />
```
Comment (Chadha93) [inline provider-ui/pages/_app.js:232]: Not required?

**Review by Chadha93 — APPROVED** [review]
Thank you @Harshit-Dandriyal. 
Is Deppendentbot not taking care of this? cc @warunicorn19

File: provider-ui/pages/_app.js (hunk 1 above)
Comment (Harshit-Dandriyal) [reply provider-ui/pages/_app.js:232]: Provider page is not rendering UI after having "@material-ui/core": "^4.12.4", probable Reason is Client-side Exception Occurred, Basically, Our CssBaseline component is now not compatible with our provider UI. I noticed that the style override via createTheme for CSSBaseline is marked as deprecated. but without including CssBaseline, Our provider UI is rendering UI and working as it should be.

Comment (Chadha93) [reply provider-ui/pages/_app.js:232]: Gotcha, thanks for confirming.

**Comment by leecalcote** [comment]
> Thank you @Harshit-Dandriyal.
> Is Deppendentbot not taking care of this? cc @warunicorn19

Dependabot tried, but didn't include the update for `CSSBaseline`, which was causing Provider UI to fail.

**Pull request merged**